__tests__/tooling/__snapshots__/operation.test.js.snap

```diff
@@ -66,10 +66,6 @@ Array [
               },
             },
             "type": "object",
-            "xml": Object {
```

Comment:
`xml` was removed from this snapshot because we're now filtering out `xml` properties from our schemas.